<ui version="4.0" >
 <class>Hawk</class>
 <widget class="QMainWindow" name="Hawk" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Hawk</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <item>
     <widget class="QTabWidget" name="workspaceTab" >
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>Tab 1</string>
       </attribute>
       <layout class="QVBoxLayout" />
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionOpen" />
    <addaction name="actionOpen_Project" />
    <addaction name="actionSave_Project" />
    <addaction name="separator" />
    <addaction name="actionExit" />
   </widget>
   <widget class="QMenu" name="menuMode" >
    <property name="title" >
     <string>Mode</string>
    </property>
    <addaction name="actionPreprocess" />
    <addaction name="actionReconstruct" />
    <addaction name="actionAnalyse" />
   </widget>
   <widget class="QMenu" name="menuColormap" >
    <property name="title" >
     <string>Colormap</string>
    </property>
    <addaction name="actionJet" />
    <addaction name="actionGrayscale" />
    <addaction name="actionTraditional" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuMode" />
   <addaction name="menuColormap" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <property name="movable" >
    <bool>true</bool>
   </property>
   <property name="allowedAreas" >
    <set>Qt::LeftToolBarArea|Qt::NoToolBarArea|Qt::TopToolBarArea</set>
   </property>
   <property name="floatable" >
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="separator" />
   <addaction name="actionPreprocess" />
   <addaction name="actionReconstruct" />
   <addaction name="actionAnalyse" />
  </widget>
  <action name="actionOpen" >
   <property name="text" >
    <string>Open Image</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionOpen_Project" >
   <property name="text" >
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionSave_Project" >
   <property name="text" >
    <string>Save Project</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit" >
   <property name="text" >
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPreprocess" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="Hawk.qrc" >:/images/32x32/configure.png</iconset>
   </property>
   <property name="text" >
    <string>Preprocess</string>
   </property>
  </action>
  <action name="actionReconstruct" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="Hawk.qrc" >:/images/32x32/exec.png</iconset>
   </property>
   <property name="text" >
    <string>Reconstruct</string>
   </property>
  </action>
  <action name="actionAnalyse" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="Hawk.qrc" >:/images/32x32/viewmag.png</iconset>
   </property>
   <property name="text" >
    <string>Analyse</string>
   </property>
  </action>
  <action name="actionJet" >
   <property name="text" >
    <string>Jet</string>
   </property>
  </action>
  <action name="actionGrayscale" >
   <property name="text" >
    <string>Greyscale</string>
   </property>
  </action>
  <action name="actionTraditional" >
   <property name="text" >
    <string>Traditional</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Hawk.qrc" />
 </resources>
 <connections/>
</ui>
